### PR TITLE
Loki doc updates

### PR DESCRIPTION
**Pull request by kenafoster**
## Reference Issues or PRs
https://github.com/nebari-dev/nebari-docs/issues/442

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [X ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by kcpevey** [comment]
This page is really directed at the sys admin user who manages deployments. I actually think the page needs to be renamed "How to setup monitoring on Nebari". Then, a separate page would show "How to access system logs (Loki) via Grafana" - which would walk folks how to actually get to the logs. 

In particular, I'd like to understand what each of the labels mean 

image

Under each of those labels, there is a variety of values. I'd like to understand (at a very high level) what each of those are (or at least the subtypes, e.g. `hub-{somehash}`)

**Comment by viniciusdc** [comment]
> Under each of those labels, there is a variety of values. I'd like to understand (at a very high level) what each of those are (or at least the subtypes, e.g. hub-{somehash})

I think it would benefit or linkig to https://grafana.com/docs/loki/latest/query/log_queries/ for an overview of how to write and understand the querying structure, and https://grafana.com/docs/grafana/latest/datasources/loki/query-editor/#choose-a-query-editing-mode for how the editor works. Also, you can get a gist of some of the "labels" there here https://kubernetes.io/docs/reference/glossary/?workload=true, it usually means what type of resource present in you k8s cluster you can target and filter logs from.

File: docs/docs/how-tos/access-logs-loki.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+![Grafana Select Loki](/img/how-tos/6_grafana-view-pod-logs.png)
+
+You can also filter by time by clicking on the time filter on top right (next to "Run query").
+
+## Common Queries
+
+To see **logs from all deployed apps**, use the label filter `container` = `notebook`.  (The main container in each deployed app pod is named `notebook`).
+
+To see **logs from a specific app**, use the `pod` selector and begin typing either the name of the user running the app or the app name to find the correct pod.  App pods are named with the convention `jupyter-[username]--[app_name]-[pod_id]`
+
```
Comment (kcpevey) [inline docs/docs/how-tos/access-logs-loki.md:42]:
```suggestion
```

moving these to the bottom on this list

File: docs/docs/how-tos/access-logs-loki.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+To see **logs from your main JupyterLab server**, again use `pod` selector and begin typing your username.  The pod name will be `jupyter-[username]`.
+
+To see **conda store** logs, use 'pod' and select one of the pods below.  Generally, user requests and access logs will be in the server pod, while logs related to environment builds will be in the worker.
+
+* `nebari-conda-store-server-[id]`
+* `nebari-conda-store-worker-[id]`
+* `nebari-conda-store-minio-[id]`
+* `nebari-conda-store-postgresql-postgresql-0`
+* `nebari-conda-store-redis-master-0`
+
```
Comment (kcpevey) [inline docs/docs/how-tos/access-logs-loki.md:52]:
```suggestion
### Deployed app logs (via jhub-apps)

If you have `jhub-apps` enabled on your deployment can you view the logs to debug deployed apps. 

To see the logs from **all deployed apps**, use the label filter `container` = `notebook`. The main container in each deployed app pod is named `notebook`.

To see **logs from a specific app**, use the `pod` label and begin typing either the name of the user running the app or the app name to find the correct pod.  App pods are named with the convention `jupyter-[username]--[app_name]-[pod_id]`.
```

**Review by kcpevey — CHANGES_REQUESTED** [review]
@kenafoster I've review the `access-logs-loki` doc here, but I need to add an image and extra text to the setup doc so rather work through suggested changes, I'll just push changes.

File: docs/docs/how-tos/setup-monitoring.md [hunk 3]
```diff
@@ -147,3 +119,28 @@ monitoring:
...
+
+End users viewing the logs will create queries in `Grafana` using `Loki` as the data source. Generally, users will query based on `label`, but its important to note that `labels` in `Grafana/Loki` are not the same as kubernetes labels. The `Grafana` labels are more accurately called _streams_ and are used to aggregate kubernetes logs from multiple services into a single "source". This allows us to configure the logging so that users can quickly access logs from multiple services with a single logical label.
+
+Loki's "labels" are an aggregation of different [targets](https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#scrape_configs) which are used to filter out logs from the [kubernets_sd](https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#kubernetes_sd_config) API endpoint. The default options are also called [streams](https://grafana.com/docs/loki/latest/get-started/labels/#understand-labels) and include labels like `pod` for collecting logs from all pods for example.
+
+For details on how to view specific logs in Loki, check out the document ["How to access system logs via Grafana"](access-logs-loki)
```
Comment (viniciusdc) [inline docs/docs/how-tos/setup-monitoring.md:137]:
```suggestion
:::note
Loki's "labels" are used to filter collections of logs from the available [kubernetes_sd](https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#kubernetes_sd_config) API endpoints, in a similar way as to how Prometheus handles metrics. These labels are configured through Promtail, which is the agent responsible for collecting and shipping logs to Loki, based on the defined [targets](https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#scrape_configs) and scraping configurations.
:::

For details on how to view specific logs in Loki, check out the document ["How to access system logs via Grafana"](access-logs-loki)
```

File: docs/docs/how-tos/monitoring.md [hunk 4]
```diff
@@ -31,34 +31,50 @@ aggregation system inspired by Prometheus. It is designed to be very cost-effect
...
 
-Click on the Log browser and select labels to search in, in this case we have selected pod:
+Select a Label to search.  In this case we are selecting 'pod':
 
 ![Grafana Select Loki](/img/how-tos/3_grafana-log-browser-pod.png)
 
-Type the pod name initials for the pod you're looking to search logs for, in this case we are looking
-for hub pod:
+Begin typing the name of the desired pod.  In this case we are looking
+for JupyterHub pod:
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply docs/docs/how-tos/monitoring.md]: I'm still wondering about this - if all users dont have access to this, then we should rework this example

[the remark this answers is not in the document]
Comment (kenafoster) [reply docs/docs/how-tos/monitoring.md]: Sorry missed this first go-around.  The only users who have access to this in Nebari's out-of-the-box configuration are those in the 'admin' keycloak group (these settings can be changed).

To me, reworking probably means just add note like:

The Explore functionality is only available to users who have `grafana_admin` permissions - a `grafana` client role in Keycloak.  In Nebari's default configuration, users in the `admin` user group will have this role.

[the remark this answers is not in the document]
Comment (kcpevey) [reply docs/docs/how-tos/monitoring.md]: ok thanks, can you add that note?

[the remark this answers is not in the document]
Comment (kenafoster) [reply docs/docs/how-tos/monitoring.md]: Yep - added!

**Review by kcpevey — APPROVED** [review]
Thanks @kenafoster! This looks great. Thanks for your patience on this!

**Pull request merged**